debian/naemon-livestatus.install

```diff
@@ -2,3 +2,5 @@
 /usr/lib/naemon/naemon-livestatus/livestatus.so
 /var/log/naemon
 /etc/naemon/module-conf.d/livestatus.cfg
+/usr/lib/python2.7/dist-packages/livestatus/__init__.py
+/usr/lib/python2.7/dist-packages/livestatus/livestatus.py
```

Comment:
Sorry my Debian package knowledge is weak, however I did figure out a way with dh_install.